polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -825,60 +858,6 @@ $loading-panel-height: 53px;
       z-index: auto;
     }
   }
-
-  // stylelint-disable selector-max-class, selector-max-combinators, selector-max-specificity -- generated by polaris-migrator DO NOT COPY
```

Comment:
Styling the sticky cell down here causes unwanted specificity that requires unnecessary redundancy. Moved :last-child to the decls for the other cells in the various interaction states.